doc/lua_api.txt

```diff
@@ -1892,7 +1892,7 @@ Call these functions only at load time!
       * The provided function should check that the position is protected by the mod
         calling this function before it prints a message, if it does, to allow for
         multiple protection mods.
-* `minetest.register_on_item_eat(func(hp_change, replace_with_item, itemstack, user, pointed_thing))`
+* `minetest.register_on_item_eat(func(hunger_change, replace_with_item, itemstack, user, pointed_thing, hp_change))`
```

Comment:
https://github.com/minetest/minetest/pull/3388/files#diff-1305560bd8befb32862f0feeefabd02eR2215

But i agree it should be made more clear ^